components/Layout/Layout.tsx

```diff
@@ -30,7 +30,10 @@ const Layout = (props: LayoutProps) => {
 				showNavMobile={showNavMobile}
 			/>
 			<div className={styles.layoutContainer}>
-				<NavPrimary activeNavLink={activeNavLink} />
+				<NavPrimary
+					activeNavLink={activeNavLink}
+					handleNavClick={handleNavClick}
```

Comment:
Hmm, handleNavClick doesn't need passing down to NavPrimary as it's only necessary for the mobile view.

Reply:
Ops, my bad. I meant to remove it from the `NavPrimary` component. It was introduced in a previous PR.